webpack.config.js

```diff
@@ -105,20 +105,26 @@ const copyPluginIndex = defaultConfig.plugins.findIndex(
 );
 
 if ( copyPluginIndex > -1 ) {
-	const blockJsonPatternIndex = defaultConfig.plugins[
-		copyPluginIndex
-	].patterns.findIndex( ( pattern ) => pattern.from === '**/block.json' );
-
-	if ( blockJsonPatternIndex > -1 ) {
-		defaultConfig.plugins[ copyPluginIndex ].patterns[
-			blockJsonPatternIndex
-		] = {
-			...defaultConfig.plugins[ copyPluginIndex ].patterns[
-				blockJsonPatternIndex
-			],
-			to: `blocks/`,
-		};
-	}
+	defaultConfig.plugins[ copyPluginIndex ].patterns.forEach(
+		( pattern, index ) => {
+			if ( pattern.from === '**/block.json' ) {
+				defaultConfig.plugins[ copyPluginIndex ].patterns[ index ] = {
+					...defaultConfig.plugins[ copyPluginIndex ].patterns[
+						index
+					],
+					context: resolve( pkg.config.coreThemeDir, 'blocks/' ),
+					to: resolve( pkg.config.coreThemeDir, 'dist/blocks/' ),
+				};
+			} else if ( pattern.from === '**/*.php' ) {
+				defaultConfig.plugins[ copyPluginIndex ].patterns[ index ] = {
+					from: '**/*.php',
+					noErrorOnMissing: true,
+					context: resolve( pkg.config.coreThemeDir, 'blocks/acf' ),
+					to: resolve( pkg.config.coreThemeDir, 'dist/blocks/acf' ),
+				};
+			}
+		}
+	);
```

Comment:
Is the underlying issue here that ACF blocks don't utilize a block.json file? Because, [ACF recommends using a block.json](https://www.advancedcustomfields.com/resources/blocks/#1-register-a-block) file for registering their blocks, just like core or other custom blocks.

Reply:
Sorry - I didn't explain this too well in the PR itself. The issue is that when we pull `block.json` over into the `dist` directory, when we set `acf.renderTemplate`, it's looking in the same folder for the file. Instead of having to write up a huge path to get back into the correct directory to read the render template, we're just pulling the render template into `dist` as well, along with the `block.json` file.

Reply:
Gotcha. I'm following now. I'm fine with this approach for Careforth. I want to stew on the overall issue a bit before committing to it for Moose, if that's okay? I'm ooo next week, but as soon as I return, let's take another look at it.  Thanks!